File: BeeCrowd/ListaDeChamada/Chamada.java
package ListaDeChamada;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Chamada {
    public static void ordena(List<String> listas){

        for(int i = 1; i < listas.size(); i++){
            String temp = listas.get(i); 
            int j = i - 1;

            while(j >= 0 && listas.get(j).compareTo(temp) > 0){

                listas.set(j + 1, listas.get(j));
                j--;
            }
            listas.set(j + 1, temp);

        }


    }

    public static void main(String[] args) {
        
        Scanner ler = new Scanner(System.in);

        List<String> lista = new ArrayList<>();

        int tam = ler.nextInt();
        int sorteado = ler.nextInt();
        

        for(int i = 0; i < tam; i++){
            String name = ler.next();
            lista.add(name);

        }

        ordena(lista);

        System.out.println(lista.get(sorteado - 1));



        ler.close();
    }
}

File: BeeCrowd/ListaDeChamada/Chamada.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

void ordena(char *array[], int tam)
{
    char temp[20];

    for (int i = 1; i < tam; i++)
    {
        strcpy(temp, array[i]);
        int j = i - 1;

        while (j >= 0 && strcmp(array[j], temp) > 0)
        {

            strcpy(array[j + 1], array[j]);
            j--;
        }
        strcpy(array[j + 1], temp);
    }
}

int main()
{

    int sorteado;
    int tam;

    scanf("%i", &tam);
    scanf("%i", &sorteado);

    char **array = (char **)malloc(tam * sizeof(char *));

    for (int i = 0; i < tam; i++)
    {

        array[i] = (char *)malloc(20 * sizeof(char));

        scanf(" %s", array[i]);

    }

    ordena(array, tam);

    printf("%s\n", array[sorteado - 1]);

    for (int i = 0; i < tam; i++)
    {
        free(array[i]);
    }
    free(array);

    return 0;
}
